Resources/Prototypes/_Goobstation/Entities/Objects/Misc/subdermal_implants.yml

```
@@ -19,3 +19,16 @@
     tags:
     - SubdermalImplant
     - HideContextMenu
+
+- type: entity
+  parent: BaseSubdermalImplant
+  id: PacifismImplant
+  name: pacifism implant
+  description: Pacifism implant, given to life sentenced prisoners or such to prevent them from committing violence.
+  categories: [ HideSpawnMenu ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)